pom.xml

```diff
@@ -73,6 +73,7 @@
     <modules>
         <module>api_validation</module>
         <module>dist</module>
+        <module>workload_profiling</module>
```

Comment:
I think we decided that we want to use a more generic name for this module and for the jar?

Reply:
I had changed the jar name to `rapids-4-spark-tools` but had kept the module name as `workload_profiling`. Sorry, got confused by this comment: https://github.com/NVIDIA/spark-rapids/pull/2402#discussion_r631316255 . Will update the module name as well.